### DISCUSSION TITLE
New Squadrons system leaves many aircraft types out

### DISCUSSION THREAD

**Discussion by ObsceneNickname** [Khopa/dcs_liberation#1827] [Q&A] [answered]
As reported in the title, the new squadrons system leaves many aircraft types out for factions that have many.

Example: BluFor Modern: There is no way to buy or assign a team for the Viggen AJS37.

However it is always a pleasure to see new updates for Liberation. It's a fantastic work

**Comment by Starfire13** [comment, marked as answer]
https://github.com/dcs-liberation/dcs_liberation/issues/1744
